From ad019c0e1f83447fd4c7bb17059a6fb65e4fe4ea Mon Sep 17 00:00:00 2001
From: temich719 <temich719@mail.ru>
Date: Sun, 20 Mar 2022 17:58:48 +0300
Subject: [PATCH] second upd of first file

---
 firstFile.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/firstFile.txt b/firstFile.txt
index 366787f..0771a74 100644
--- a/firstFile.txt
+++ b/firstFile.txt
@@ -1,5 +1,5 @@
-1
-2
+upd3
+upd4
 3
 4
 5
-- 
2.33.1.windows.1

